### PR TITLE
Bring back jaegerServiceEndpoint ENV for backward compatibility

### PR DESCRIPTION
**Pull request by kwiatekus**
**Related issue(s)**
Fixes https://github.com/kyma-project/kyma/issues/15790

### CODE REVIEW

**Review by pPrecel — COMMENTED** [review]
Looks ok but right now we have a kind of schizophrenia because of two same ENVs with different names. We should add any info that this is expected and we will change it after release. (comment, `// TODO: ...` or maybe even ticket)

**Review by pPrecel — DISMISSED** [review]

**Review by pPrecel — DISMISSED** [review]

File: docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md [hunk 1]
```diff
@@ -19,8 +19,12 @@ Every runtime provides its own unique environment configuration which can be rea
 | **KUBELESS_INSTALL_VOLUME** | `/kubeless` | Full path to volume mount with users source code. |
 | **FUNC_RUNTIME** | | The name of the actual runtime. Possible values: `python39`, `nodejs12`, `nodejs14`, `nodejs16`. |
 | **TRACE_COLLECTOR_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:4318/v1/traces` | Full address of the Open-Telemetry Trace Collector. |
+| **JAEGER_SERVICE_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:14268/api/traces` | **Deprecated in 2.8** Full address of the Jaeger service. |
 | **PUBLISHER_PROXY_ADDRESS** | `http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish` | Full address of the Publisher Proxy service. |
 
+> **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 are using OpenTelemetry protocol compliant endpoint as defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
```
Comment (grego952) [inline docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md:25]:
```suggestion
> **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 use the OpenTelemetry protocol-compliant endpoint defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
```

**Review by grego952 — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

File: docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md (hunk 1 above)
Comment (grego952) [inline docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md:25]:
```suggestion
> **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 use OpenTelemetry protocol-compliant endpoint as defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
```

File: docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md [hunk 2]
```diff
@@ -19,8 +19,12 @@ Every runtime provides its own unique environment configuration which can be rea
 | **KUBELESS_INSTALL_VOLUME** | `/kubeless` | Full path to volume mount with users source code. |
 | **FUNC_RUNTIME** | | The name of the actual runtime. Possible values: `python39`, `nodejs12`, `nodejs14`, `nodejs16`. |
 | **TRACE_COLLECTOR_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:4318/v1/traces` | Full address of the Open-Telemetry Trace Collector. |
+| **JAEGER_SERVICE_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:14268/api/traces` | **Deprecated in 2.8** Full address of the Jaeger service. |
 | **PUBLISHER_PROXY_ADDRESS** | `http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish` | Full address of the Publisher Proxy service. |
 
+> **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 use OpenTelemetry protocol-compliant endpoint as defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
```
Comment (grego952) [inline docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md:25]:
```suggestion
> **NOTE:** TRACE_COLLECTOR_ENDPOINT replaced JAEGER_SERVICE_ENDPOINT in Kyma 2.8. Functions built (or re-built) after 2.8 use OpenTelemetry protocol-compliant endpoint, as defined by TRACE_COLLECTOR_ENDPOINT. JAEGER_SERVICE_ENDPOINT will be removed in Kyma 2.10.
```

**Review by grego952 — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Pull request merged**